readme.md

```diff
@@ -200,6 +200,9 @@ The time complexity of an algorithm estimates how much time the algorithm will u
 ### [Space Complexity](Algorithms/Space%20Complexity/readme.md)
 Space complexity refers to the total amount of memory space used by an algorithm/program, including the space of input values for execution. Calculate the space occupied by variables in an algorithm/program to determine space complexity.
 
+### [Space Complexity](Algorithms/Space%20Complexity/readme.md)
```

Comment:
Space complexity section in two places.

Reply:
Corrected it now
Please review it.
Now only AVL Tree is added